Rakefile

```diff
@@ -6,3 +6,25 @@ require 'bugsnag-capistrano/tasks'
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+namespace :db do
+  task :schemafile => :load_config do
+    db_config = ActiveRecord::Base.configurations.configs_for(env_name: ActiveRecord::Tasks::DatabaseTasks.env).first
+    ActiveRecord::Tasks::PostgreSQLDatabaseTasks.new(db_config).set_schema_file
+  end
+end
+
+ActiveRecord::Tasks::PostgreSQLDatabaseTasks.prepend(Module.new do
+  def set_schema_file
+    clear_active_connections!
+    establish_master_connection
+    server_version = connection.select_value("SHOW server_version").to_i
+    suffix = server_version < 11 ? '' : '-12'
```

Comment:
If it's psql 11, we still append `-12`? I guess the semantics then is "schema is meant for psql v12", even if the customer uses with psql 11.

Reply:
Well because of the version used in Dockerfile and other scripts ... I do not know I would like just to migrate to 12 and not having backward compatibility